dask/distributed/tests/test_distributed.py

```diff
@@ -2,16 +2,20 @@
 
 import time
 from subprocess import Popen, PIPE
+from distutils.version import StrictVersion
```

Comment:
- They throw a warning, not an error
- distutils is stdlib, and has code made for version comparisons.  I don't see the problem with using this.